server/datastore/mysql/app_configs.go

```
@@ -0,0 +1,53 @@
+package mysql
+
+import (
+	"database/sql"
+
+	"github.com/kolide/kolide-ose/server/kolide"
+)
+
+func (d *Datastore) NewAppConfig(info *kolide.AppConfig) (*kolide.AppConfig, error) {
+	var (
+		err    error
+		result sql.Result
+	)
+
+	err = d.db.Get(info, "SELECT * FROM app_configs LIMIT 1")
+	switch err {
+	case sql.ErrNoRows:
+		result, err = d.db.Exec(
+			"INSERT INTO app_configs (org_name, org_logo_url, kolide_server_url) VALUES (?, ?, ?)",
+			info.OrgName, info.OrgLogoURL, info.KolideServerURL,
+		)
+
+		if err != nil {
+			return nil, err
+		}
+
+		info.ID, _ = result.LastInsertId()
```

should we be ignoring the `err` here?

---

The only time LastInsertId will return an error is if we switch to another RDBMS that doesn't return an ID on inserts, say Postgres for example.  In that case we'll need to remove the whole line of code, so in my judgement, it's OK to ignore the err.